community/shell/src/main/java/org/neo4j/shell/kernel/GraphDatabaseShellServer.java

```diff
@@ -208,6 +211,13 @@ private static GraphDatabaseAPI instantiateGraphDb( GraphDatabaseFactory factory
         {
             builder.loadPropertiesFromFile( configFileOrNull );
         }
+        // Explicitly disable all connectors since we're starting an embedded database for the shell client (only!)
+        // This only deals with Bolt since it's the only connector type to start from a kernel extension
+        Config config = builder.config();
+        config.view( enumerate( GraphDatabaseSettings.Connector.class ) )
+                .map( GraphDatabaseSettings.BoltConnector::new )
+                .filter( ( connConfig ) -> config.get( connConfig.enabled ) )
+                .forEach( connector -> builder.setConfig( connector.enabled, Settings.FALSE ) );
```

Comment:
This is where we use `GraphDatabaseBuilder.config()`, to iterate over all configured (bolt) connectors, and setting them to **not enabled**.